src/react-components/src/__tests__/components/EditionCard-test.tsx

```diff
@@ -84,5 +94,15 @@ describe("EditionCard", () => {
     expect(card.find("h3")).to.have.lengthOf(1);
     expect(card.find("h3").find("a")).to.have.lengthOf(1);
     expect(card.find("img")).to.have.lengthOf(1);
+    expect(card.find("#pub-span").text()).to.equal("Publisher");
+    expect(card.find("a#licenceId").text()).to.equal("License");
+  });
+
+  it("Passes call-to-action links if it is given as span", () => {
```

Comment:
Just to verify, what get's passed as `readOnlineLink` and `downloadLink` won't actually be links, right? So if it's passed as a span, it'll just be a span but not a  call-to-action "link"?

Reply:
sigh. yeah.  The name is now wrong, but I don't know if now's a good time to make this breaking change.

Reply:
I would write a "TODO" comment for the props in the component so you can be reminded to update it later on.